gems/pending/spec/postgres_ha_admin/failover_monitor_spec.rb

```diff
@@ -15,9 +15,9 @@
   end
 
   let(:failover_monitor) do
-    expect(PostgresHaAdmin::DatabaseYml).to receive(:new).and_return(db_yml)
-    expect(PostgresHaAdmin::FailoverDatabases).to receive(:new).and_return(failover_db)
-    failover_instance = described_class.new('', '', @logger_file.path, 'test')
+    allow(PostgresHaAdmin::DatabaseYml).to receive(:new).and_return(db_yml)
+    allow(PostgresHaAdmin::FailoverDatabases).to receive(:new).and_return(failover_db)
```

Comment:
Creation of another instance of ```FailoverMonitor``` was added in new expectation, and now there would be two invocations of ```DatabaseYml.new``` when instance ```FailoverMonitor``` created.  I was thinking it would look misleading if we require exactly  2 invocation in above lines.